### ISSUE TITLE
How to set specific origin?

### ISSUE THREAD

**Issue by hopewise** [palanik/lumen-cors#4]
I've installed the package, but how would I use it to accept access from specific domains?

**Comment by puneetthakur66** [comment]
@hopewise I know it is late, just writing this so that it could help someone else.
You can add this in the settings array

protected $settings = array(
        'origin' => 'https://particulardomain.com    // Wide Open!
        'allowMethods' => 'GET,HEAD,PUT,POST,DELETE,PATCH,OPTIONS',
    );

**Comment by danielperaza** [comment]
I don't think this is a good solution, since if you install the package via composer, you would need to modify the original package's contents and those changes can be later lost, if you delete your `vendors` folder, which should not be under version control.
